map.html

```diff
@@ -261,7 +261,14 @@ <h3 class="sidebar__town">Everett</h3>
 
 
 
-    <div class="metrocommon">The MetroCommon section</div>
+    <section class="metrocommon">
+      <div class="metrocommon__left"></div>
```

Comment:
Consider changing this from a `div` with a background to an `img` tag and refactoring the class names to be more descriptive (like `metrocommon__image` and `metrocommon___text`)

Reply:
Sure!